RecoTracker/IterativeTracking/python/PixelLessStep_cff.py

```diff
@@ -321,6 +321,8 @@
     seeds = 'pixelLessStepSeeds',
     mkFitSeeds = 'pixelLessStepTrackCandidatesMkFitSeeds',
     tracks = 'pixelLessStepTrackCandidatesMkFit',
+    algo = 9,
```

Comment:
Maybe can use `int(TrackBase::TrackAlgorithm::pixelLessStep)`, that should be equivalent? Just to avoid other numbers flying around.

Reply:
yes, I have to set a working point for the cut in the config and have a flag under a reasonable name.

can int(TrackBase::TrackAlgorithm::pixelLessStep) point me to different iterations in case this code is run for multiple iterations?

Reply:
> can int(TrackBase::TrackAlgorithm::pixelLessStep) point me to different iterations in case this code is run for multiple iterations?

Yes, can use this for all iterations, as per https://github.com/cms-sw/cmssw/blob/master/DataFormats/TrackReco/interface/TrackBase.h#L89-L142

Reply:
but can I ask the track "what is your iteration?". anyway, don't worry, I'll try to find out if it's possible

Reply:
I should use reco::TrackBase::algoByName suggested by @slava77 a while ago most likely, to get the name from one of the producer input strings